### ISSUE TITLE
Why patch version not allowed?

### ISSUE THREAD

**Issue by Maxim-Mazurok** [microsoft/definitelytyped-header-parser#30]
I don't understand why patch version is not allowed.

As far as I can tell, it was like that https://github.com/microsoft/definitelytyped-header-parser/blob/808e8a32f4c5150318bf5d9a164b881265157c58/index.ts#L245 from the start (https://github.com/microsoft/definitelytyped-header-parser/commit/87d8817ddfc65331818b8a292a6e2b6ea76990ef) as written by @andy-ms

I understand that it probably won't change, but I would like to know why this decision was made? I think that I'm missing a point...

The reason why I'm asking is because we are working on [google-api-typings-generator](https://github.com/Maxim-Mazurok/google-api-typings-generator) and our users could [benefit greatly](https://github.com/Maxim-Mazurok/google-api-typings-generator/issues/15#issuecomment-572714972) if patch versions could be allowed.
Basically, we generate typings for google APIs, they have major and minor version, but they also have revisions within the same `x.x` versions. So, we were thinking about using patch versions to indicate typings for new revisions.

Any thoughts or explanation regarding this matter would be greatly appreciated!

**Comment by Maxim-Mazurok** [comment]
It seems to be because DT automatically increases patch version when releasing type definitions to NPM.
